src/bin/common/parse_command.ml

```diff
@@ -37,13 +37,21 @@ exception Error of bool * string
 (* Exception used to exit with corresponding retcode *)
 exception Exit_parse_command of int
 
+let pretty_error msg =
+  match Options.get_output_format () with
+  | Smtlib2 -> Format.asprintf "; %s" msg
+  | _ -> msg
+
```

Comment:
This is a best effort prettyfier. Also, I actually did that because of a test that added a ';' on an error message (when a plugin fails to be loaded). I can update the test, but I was not sure of the specification of the output message for parsing commands. I agree with you though, it would be better to just not consider parsing command errors as SMTLIB2 errors. I just did not want to argue on why I removes the ';' on the test.

Reply:
Yes, removing the `;` should be fine in that case -- it looks out of place in the test and was probably kept there as a historical accident. This is a PR for cleaning up error strategies & error messages; it is expected that some error messages may slightly change :)

Reply:
I'll revert these changes then :)